GhidraPython/goresym_rename.py

```diff
@@ -0,0 +1,174 @@
+#This script labels Go stripped binaries in Ghidra based on GoReSym output
```

Comment:
Is this script meant to be run under Jython/python2.7 in Ghidra or under Python 3 via Ghidrathon or it doesn't matter? I think we should document this somewhere.

Reply:
also recommend a short comment at the top explaining what is expected as input and what will happen as output.